test/fixtures/es-module-loaders/builtin-named-exports-loader.mjs

```diff
@@ -1,23 +1,62 @@
 import module from 'module';
 
-export function getFormat(url, context, defaultGetFormat) {
-  if (module.builtinModules.includes(url)) {
+const GET_BUILTIN = `$__get_builtin_hole_${Date.now()}`;
+
+export function getGlobalPreloadCode() {
```

Comment:
The only reason for needing this is because this hook is messing with the builtin `module` module. Otherwise the generated code could `import {createRequire} from 'module'` without needing a global.

Reply:
If we are going to push users into this hook now, then I think we should aim to work on this usability first - you shouldn't need to define a secret global for the common use case...